client/public/locales/fr.json

```diff
@@ -7,13 +7,13 @@
     "realizedBy": "Fait par {0}"
   },
   "menu": {
-    "devices": "appareils",
-    "home": "home",
-    "blockly": "blockly"
+    "devices": "Appareils",
```

Comment:
I think it will better to write in lowercase and then use css attribute "capitalize"